examples/plot_custom_colours.py

```diff
@@ -0,0 +1,72 @@
+"""
+=================================
+Plot RGBA values on brain surface
+=================================
+
+"""
+import nibabel
+import numpy as np
+import matplotlib.pyplot as plt
+from mayavi import mlab
+from tvtk.api import tvtk
+from tvtk.common import configure_input_data
+from surfer.utils import smoothing_matrix, mesh_edges
+from surfer import Brain
+
+print(__doc__)
+
+###############################################################################
+# define helper functions
+
+def norm(x):
+	'''
+	Normalise array betweeen 0-1
+	'''
+	return (x - np.min(x)) / (np.max(x) - np.min(x))
+
+###############################################################################
+# load surface
+
+surf_fname = '/Users/lauragwilliams/Documents/programming/PySurfer/examples/example_data/lh.white'
```

Comment:
This won't work when building the docs on travis or other people's computers, but you can build a path using `os.environ["SUBJECTS_DIR"]`